src/assets/wise5/common/object/object.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { copy } from './object';
+
+describe('copy()', () => {
+  it('should copy arrays and objects', () => {
+    const arr = [1, 2, 3];
+    expect(copy(arr)).toEqual(arr);
+    const obj = { name: 'WISE', address: 'Berkeley' };
+    expect(copy(obj)).toEqual(obj);
+  });
+  it('should handle null and undefined inputs', () => {
+    expect(copy(null)).toEqual(null);
+    expect(copy(undefined)).toEqual(undefined);
+  });
+});
```

Comment:
I think these 2 tests should be broken into 4 tests. I think the array and object test should make a copy and then modify the original object and make sure the new copy does not receive that modification.